src/niswitch/examples/niswitch_channel_properties.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+import argparse
+import niswitch
+import sys
+
+parser = argparse.ArgumentParser(description='TODO: write something.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+args = parser.parse_args()
+
+try:
+    with niswitch.Session(args.name) as session:
+        with session.channel("Marcos") as chan:
+            print('channel {0} bandwidth is {1}'.format(chan, chan.bandwidth))
```

Comment:
If you are going to use the repeated capability object in format, then you should add with \_\_format\_\_ or \_\_repr\_\_ or \_\_str\_\_ to the _RepeatedCapabilities class

Reply:
I had __repr__() for a bit, but there were some added trickiness so I removed and filed #328.
Need to delete this example.